src/response.js

```diff
@@ -21,7 +21,8 @@ export default class Response extends Body {
 	constructor(body = null, options = {}) {
 		super(body, options);
 
-		const status = options.status || 200;
+		// TODO Replace with nullish coalescing operator (??) when support for Node.js < 14 is dropped
+		const status = options.status !== null && options.status !== undefined ? options.status : 200;
```

Comment:
@tkrotoff Please use `options.status != null` for this comparison, and make sure to put the eslint-disable directive on the preceding line: `// eslint-disable-line no-eq-null, eqeqeq`

Reply:
Done.
```JS
// eslint-disable-next-line no-eq-null, eqeqeq, no-negated-condition
const status = options.status != null ? options.status : 200;
```

Reply:
@tkrotoff Sorry, one more thing.  Since Node.js 12 will be supported for at least 16 more months, please take out the TODO comment.  I'm sure we'll do a big push to update those kinds of changes at that point.

Reply:
Done.